src/js/actions/tools.js

```diff
@@ -65,6 +65,8 @@ define(function (require, exports) {
      */
     var _currentTransformPolicyID = null;
 
+    var directSelectToolID = "superselectVector";
```

Comment:
There isn't a super satisfying way to deal with shared keys like this. I think this is fine, and I think it is also fine to just use an inline string. You could make it a property of a store, but then you'd have to look up that property with another string... So, whatev!